lib/ts/recipe/session/sessionClass.ts

```diff
@@ -121,4 +134,41 @@ export default class Session implements SessionContainerInterface {
             );
         }
     };
+
+    getTimeCreated = async (): Promise<number> => {
+        try {
+            let querier = Querier.getNewInstanceOrThrowError(SuperTokens.getInstanceOrThrowError().isInServerlessEnv);
+            let apiVersion = await querier.getAPIVersion();
+
+            if (maxVersion(apiVersion, "2.7") === "2.7") {
+                throw new Error("Please use core version >= 3.5 to call this function.");
+            }
+
+            return (await SessionFunctions.getSessionInformation(this.recipeImplementation, this.sessionHandle))
```

Comment:
Checking of `apiVersion` (or throwing error) should happen inside `SessionFunctions.getSessionInformation`.. so that this code is not repeated whenever `SessionFunctions.getSessionInformation` is used.

Reply:
Done